build_artifacts.py

```diff
@@ -411,10 +431,10 @@ def generate_json_resources(self):
         """
```

Comment:
Above where you do `for key, value in response.json().items(): ...`, since `key` is unused, could you iterate over `response.json().values()` instead?